mmv1/third_party/terraform/services/securitycenter/resource_scc_folder_big_query_export_config_test.go

```diff
@@ -0,0 +1,131 @@
+package securitycenter_test
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/hashicorp/terraform-plugin-testing/plancheck"
+	"github.com/hashicorp/terraform-provider-google/google/acctest"
+	"github.com/hashicorp/terraform-provider-google/google/envvar"
+)
+
+func TestAccSecurityCenterFolderBigQueryExportConfig_update(t *testing.T) {
+	t.Parallel()
+
+	randomSuffix := acctest.RandString(t, 10)
+	dataset_id := "tf_test_" + randomSuffix
+	dataset_id2 := "tf_test_" + randomSuffix
```

Comment:
In principle this works, but right now the values here are both the same, so it won't actually trigger a change to the dataset resource.

```suggestion
	dataset_id := "tf_test_" + randomSuffix
	dataset_id2 := "tf_test_" + acctest.RandString(t, 10)
```

Or alternatively:

```suggestion
	dataset_id := "tf_test_" + randomSuffix
	dataset_id2 := dataset_id + "2"
```

Reply:
Updated.